packages/monitoring-system/metrics-dashboard/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: gcr.io/cloud-builders/gcloud
+    id: "get-firebase-token"
+    waitFor: ["-"]
+    entrypoint: 'bash'
+    args: 
+      - '-c'
+      - "gcloud secrets versions access latest --secret=firebase-token-metrics-dashboard --format='get(payload.data)' | tr '_-' '/+' | base64 -d > decrypted-data.txt"
+
+  - name: gcr.io/cloud-builders/npm
+    id: "deploy-dashboard"
+    waitFor: ["get-firebase-token"]
+    entrypoint: bash
+    args:
+      - "-e"
+      - "./packages/monitoring-system/metrics-dashboard/scripts/deploy.sh  $(cat decrypted-data.txt)"
```

Comment:
I'd make the argument the file and not the actual secret. Rather not provide raw credentials via command line options (as well as if there are weird characters).

Additionally, you'll probably need to quote the value if you're using it on the command line